Packs/RiskIQDigitalFootprint/Integrations/RiskIQDigitalFootprint/RiskIQDigitalFootprint.py

```diff
@@ -0,0 +1,2451 @@
+                                                      'Self Hosted Resource, ThirdParty Hosted Resource.'
+                                                      ' This argument supports a single value only.',
+    'INVALID_BOOLEAN_VALUE_ERROR': 'The given value for {0} argument is invalid. Valid values: true, false.'
+                                   ' This argument supports a single value only.',
+    'NO_RECORDS_FOUND': 'No {0} were found for the given argument(s).',
+    'INVALID_DATE_FORMAT_ERROR': 'The given value for date is invalid. The accepted format for date is YYYY-MM-DD.'
+                                 ' This argument supports a single value only.',
+    'INVALID_RANGE_ERROR': 'The given value for range is invalid. Valid values: 1, 7, 30.'
+                           ' This argument supports a single value only.',
+    'INVALID_RANGE_ERROR_FOR_ASSET_DETAIL_TYPE': 'The given value for range is invalid. '
+                                                 'Only single day changes can be shown for {0} type. Valid value: 1.'
+                                                 ' This argument supports a single value only.',
+    'INVALID_ASSET_TYPE_MEASURE': 'The given value for measure(type of change) is invalid.'
+                                  ' Valid options are Added or Removed. This argument supports a single value only.',
+    'INVALID_ASSET_DETAIL_TYPE_MEASURE': 'The given value for measure(type of change) is invalid.'
+                                         ' Valid options are Added or Changed.'
+                                         ' This argument supports a single value only.',
+    'ASSET_PAYLOAD_MISSING_KEYS_ERROR': 'Required keys for {} asset(s) are {}. One or more of them are not present'
+                                        ' in the asset JSON.',
+    'INVALID_ARGUMENTS_ADD_ASSET': 'Argument asset_json cannot be used with other arguments except fail_on_error.',
+    'INVALID_ARGUMENTS_UPDATE_ASSET': 'At least one property argument should have a value in order to update the'
+                                      ' asset. The property arguments are: state, priority, removed_state, '
+                                      'brand, organization, tag and enterprise.',
+    'REQUIRED_ARGUMENTS_FOR_ASSET': 'Argument asset_json or arguments name and type are required to {0} an asset.',
+    'INVALID_ASSET_STATE_ERROR': 'The given value for state is invalid. '
+                                 'Valid States: Candidate, Approved Inventory, Requires Investigation, '
+                                 'Dependencies, Monitor Only. This argument supports a single value only.',
+    'INVALID_ASSET_PRIORITY_ERROR': 'The given value for priority is invalid. Valid Priority levels: High, Medium,'
+                                    ' Low, None. This argument supports a single value only.',
+    'INVALID_REMOVED_STATE_ERROR': 'The given value for removed state is invalid. Valid value: Dismissed.'
```

Comment:
@avidan-H We have kept this argument as a list of options considering the scenario if there can be more than one potential value in the future for it so that the user can change it.